scss/_base_button.scss

```
@@ -100,15 +100,15 @@
     width: auto;
 
     & [class*='p-icon'] {
-      margin-left: $sph-inner--small;
-      margin-right: $sph-inner--small;
+      margin-left: $sph--small;
+      margin-right: $sph--small;
 
       &:first-child {
-        margin-left: -#{$sph-inner--small};
+        margin-left: -#{$sph--small};
```

I guess the interpolation here is not needed:

```suggestion
        margin-left: -$sph--small;
```